python/README.rst

```diff
@@ -2,7 +2,7 @@
 Python API and CLI for Nessie
 =============================
 
-Nessie is a system to provide Git like capability for Iceberg Tables, Delta Lake Tables, Hive Tables and Sql Views.
+Nessie is a system to provide Git like capability for Iceberg Tables, Delta Lake Tables, Hive Tables and Iceberg Views.
 
```

Comment:
"Nessie is a system to provide Git like capability for Iceberg Tables and Views and Delta Lake Tables."